### PR TITLE
Feature/framer motion

### PR DESCRIPTION
**Pull request by JCourtney22**
Icons have framer motion and are coloured according to the nav bar colour.

### CODE REVIEW

File: src/components/BookList/index.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ import Col from 'react-bootstrap/Col'
 import Row from 'react-bootstrap/Row'
 
 import Book from '../Book'
+// Import Font Awesome
```
Comment (JonHarrison) [inline src/components/BookList/index.js]: Do these need to be add here with the Book container ?

File: src/components/Book/index.js [hunk 2]
```diff
@@ -91,12 +95,22 @@ const Book = ({item}) => {
                 </Card.Text>
             </Card.Body>
             <div className="card-book-selectors">
-                <div onClick={() => addBook(item)}>
```
Comment (JonHarrison) [inline src/components/Book/index.js:94]: Can you put the onClick bit back in :)

Comment (JCourtney22) [reply src/components/Book/index.js:94]: Ooops! Must have accidentally taken it out when mergin - sorry!

File: src/components/BookList/index.js (hunk 1 above)
Comment (JCourtney22) [reply src/components/BookList/index.js]: Again, my fault, forgot to go back to it after mergin. Will take it out.

**Review by JonHarrison — APPROVED** [review]

**Pull request merged**